src/scripts/build/nimbusTypes.js

```diff
@@ -82,7 +82,9 @@ function getFallbackObject(nimbusConfig) {
         nimbusConfig.features[featureId].variables,
       );
       const variableFallbackDefs = variableNames.map((variableName) => {
-        return `    "${variableName}": ${nimbusConfig.features[featureId].variables[variableName].default},\n`;
+        const variableValue =
+          nimbusConfig.features[featureId].variables[variableName].default;
+        return `    "${variableName}": ${typeof variableValue === "string" ? `"${variableValue}"` : variableValue},\n`;
```

Comment:
This change was needed, because the objects of `defaultExperimentData` in the generated `experiments.ts` file had an invalid syntax for string values.